src/js/common/helpers/api.js

```diff
@@ -18,11 +18,14 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
   const defaultSettings = {
     method: 'GET',
     headers: {
-      Authorization: `Token token=${sessionStorage.userToken}`,
       'X-Key-Inflection': 'camel'
     }
   };
 
+  if (sessionStorage.userToken) {
+    defaultSettings.headers.Authorization = `Token token=${sessionStorage.userToken}`;
+  }
+
```

Comment:
The way this was before we would see `Token token=undefined` and that didn't work well with the mock API